Added Release Notes 6.5.0 (#75)
owncloud/docs-main: modules/ROOT/pages/ocis_release_notes.adoc

toc::[]

== Infinite Scale 6.5.0 (Rolling)

IMPORTANT: This is a Rolling Release. Please check the {release-types-url}[documentation] to see if this release type is right for your use case.

> 

Reviewer: ```suggestion

Refer to the source and the full change log for a list of bug fixes and changes at {ocis-releases-url}/v6.5.0[GitHub, window=_blank].
```

[discrete]
=== Public and Private Links in the Sharing Panel

The sharing panel has been redesigned to separate links into two categories: Permanent Link and Public Links.

> - *Permanent Link*: A fixed link for internal use that works as a pointer for people with existing access. Use it ex. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved.

Reviewer 2: ```suggestion
- *Permanent Link*: A fixed link for internal use that works as a pointer for people with existing access. Use it e.g. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved.
```

Reviewer 2: Could you also add something like `The permanent link is located by the space memberships and people shares.`

> - *Permanent Link*: A fixed link for internal use that works as a pointer for people with existing access. Use it e.g. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved. The permanent link is located by the space memberships and people shares.
> 
> - *Public Links*: These links grant access to external people outside your organization. Public links are password-protected by default, ensuring secure file sharing with non-members. Password protection can be turned off by the system administrator.
> 

Reviewer: ```suggestion
- *Permanent Link*: +
A fixed link for internal use that works as a pointer for people with existing access. Use it e.g. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved. The permanent link is located by the space memberships and people shares.

- *Public Links*: +
These links grant access to external people outside your organization. Public links are password-protected by default, ensuring secure file sharing with non-members. Password protection can be turned off by the system administrator.
```
Improves readbility

[discrete]

> === Copy "Permanent Link" by default

Reviewer: ```suggestion
=== Copy "Permanent Link" by Default
```

Copying a link in the file list now provides the permanent link with a single click. Permanent links work only for people with existing access, ensuring access is secure by default while offering a faster way to share and point people to files.

[discrete]

> === Public Link: Copy with or without password to clipboard  

Reviewer: ```suggestion
=== Public Link: Copy With or Without Password to Clipboard  
```

You can now copy a public link to your clipboard with or without a password, giving you more control over file sharing.

[discrete]

> === Removed link types: "Can upload" and "Invited people"

Reviewer: ```suggestion
=== Removed Link Types: "Can upload" and "Invited people"
```

> The “Can upload” and “Invited people” link types have been removed to simplify sharing options. The functionality of the “Invited people” link is still available through the “Permanent link” option.

Reviewer 2: Please add something like `Existing links of those types will continue to work. It is not possible to create new public links with these types though.`

[discrete]
=== "Save as" in the app top bar

> A “Save As” option is now available in the app top bar for ex. markdown or plaintext files. You can easily save your document to the desired location without leaving the editor.

Reviewer 2: ```suggestion
A “Save As” option is now available in the app top bar for e.g. markdown or plaintext files. You can easily save your document to the desired location without leaving the editor.
```

[discrete]
=== Right Sidebar Transition

The right sidebar transition has been optimized for smoother and more natural animation, enhancing the overall user experience.

[discrete]

> === Version Information In Right Sidebar

Reviewer 2: ```suggestion
=== Version Information In Left Sidebar
```

> === Version Information In Left Sidebar

Reviewer: ```suggestion
=== Version Information in Left Sidebar
```

> The version information for Infinite Scale and the Web UI is now displayed in the bottom left sidebar. This enhancement ensures that users can easily identify the versions they are using at any time, simplifying the process for support requests.

Reviewer 2: ```suggestion
The version information for Infinite Scale and the Web UI is now displayed in the bottom of the left sidebar. This enhancement ensures that users can easily identify the software versions they are using at any time, simplifying the process for support requests.
```

[discrete]

> === Build your own Web Application: Web App Boilerplate

Reviewer: ```suggestion
=== Build Your own Web Application: Web App Boilerplate
```

toc::[]

== Infinite Scale 6.5.0 (Rolling)

IMPORTANT: This is a Rolling Release. Please check the {release-types-url}[documentation] to see if this release type is right for your use case.

Refer to the source and the full change log for a list of bug fixes and changes at {ocis-releases-url}/v6.5.0[GitHub, window=_blank].

[discrete]

> === Public and Private Links in the Sharing Panel

Reviewer 3: You call them Public and Private in the heading, but the text uses Permanent and Public.

Edit the heading, or add some explanation that Private = Permanent

> The sharing panel has been redesigned to separate links into two categories: Permanent Link and Public Links.

Reviewer 3: ```suggestion
The sharing panel has been redesigned to separate links into two categories: Permanent Links and Public Links.
```
